src/Tribe/Views/V2/Utils/Stack.php

```diff
@@ -349,7 +349,7 @@ protected function normalize_stack() {
 	protected function filter_stack_event( $event ) {
 		$post = tribe_get_event( $event );
 
-		$keep = $post instanceof \WP_Post && ( ! empty( $post->multiday ) || ! empty( $post->all_day ) );
+		$keep = $post instanceof \WP_Post && ( ( ! empty( $post->multiday ) && $post->multiday > 1 ) || ! empty( $post->all_day ) );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 132 characters

<sub>hash: bbb008497d8eae193fe8014a5fb2b97b</sub>